pkg/tso/tso.go

```diff
@@ -167,9 +168,11 @@ func (t *timestampOracle) SyncTimestamp(leadership *election.Leadership) error {
 		return err
 	}
 	lastSavedTime := t.getLastSavedTime()
-	// If `lastSavedTime` is not zero, it means that the `timestampOracle` has already been initialized
-	// before, so we could safely skip the sync if `lastSavedTime` is equal to `last`.
-	if lastSavedTime != typeutil.ZeroTime && typeutil.SubRealTimeByWallClock(lastSavedTime, last) == 0 {
+	// If `last` and `lastSavedTime` are the same and both not zero, it means that the
+	// timestamp has been synchronized before. We could skip the synchronization safely.
+	if last != typeutil.ZeroTime && lastSavedTime != typeutil.ZeroTime &&
+		typeutil.SubRealTimeByWallClock(lastSavedTime, last) == 0 {
+		log.Info("skip sync timestamp", zap.Uint32("keyspace-group-id", t.keyspaceGroupID))
```

Comment:
Will the keyspace group id be printed in non serverless environment?

Reply:
I want to retain this field to help us better debug, but in order not to trouble users in the OP environment so much, I have hidden the keyspace-related words. Do you think it is okay?

Reply:
How about using CondUint32

Reply:
Nice suggestion. Addressed.